src/index.html

```diff
@@ -6,9 +6,32 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      </div>
+      <nav class="nav">
+        <ul class="menu">
+          <li class="nav_list"><a href="#" class="item is-active">Apple</a></li>
```

Comment:
```suggestion
          <li class="nav_list">
            <a href="#" class="item is-active">Apple</a>
           </li>
```

better to start tag on new line, fix it everywhere